tests/migrations/test_operations.py

```diff
@@ -1379,6 +1379,54 @@ def test_add_field(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(sorted(definition[2]), ["field", "model_name", "name"])
 
+    @skipUnlessDBFeature("supports_stored_generated_columns")
+    def _test_add_generate_field(self):
```

Comment:
Have we confirmed that this test even run since it has a `_` prefix? I thought `unittest` only picked up method with a `test` prefix.